src/content/projects_content.json

```diff
@@ -0,0 +1,179 @@
+{
+    "sections": [
+      {
+        "title": "Projects",
+        "cards": [
+          {
+            "title": "Mentorship Android",
+            "description": "Android client for Mentorship System - an application that allows women in tech to mentor each other, on career development topics, through 1:1 relations for a certain period.",
+            "labels": [
+              "javascript",
+              "mvvm"
+            ],
+            "badges": [
+              {
+                "title": "zulip",
+                "link": "https://anitab-org.zulipchat.com/login/#narrow/stream/222534-mentorship-system"
+              },
+              {
+                "title": "github",
+                "link": "https://github.com/anitab-org/mentorship-android"
+              }
+            ]
+          },
+          
+          {
+            "title": "Mentorship Backend",
+            "description": "Backend REST API for Mentorship System - an application that allows women in tech to mentor each other, on career development topics, through 1:1 relations for a certain period.",
+            "labels": [
+              "javascript",
+              "mvvm"
+            ],
+            "badges": [
+              {
+                "title": "zulip",
+                "link": "https://anitab-org.zulipchat.com/login/#narrow/stream/222534-mentorship-system"
+              },
+              {
+                "title": "github",
+                "link": "https://github.com/anitab-org/mentorship-backend"
+              }
+            ]
+          },
+
+          {
+            "title": "Mentorship iOS",
+            "description": "iOS application for Mentorship System - an application that matches women in tech to mentor each other, on career development, through 1:1 relations during a certain period of time.",
+            "labels": [
+              "javascript",
+              "mvvm"
+            ],
+            "badges": [
+              {
+                "title": "zulip",
+                "link": "https://anitab-org.zulipchat.com/login/#narrow/stream/222534-mentorship-system"
```

Comment:
we need own content not link to Zulip, also when we use images we need to keep them in JS file and add require(relative_path), so it can be treated as static image.

Reply:
@annabauza , would be very helpful if you could elaborate, I didn't get which images you meant, I added Zulip links since the title was Zulip in the template you had told to refer in the same issue.

Reply:
@gaurivn can you please review my PR, maybe this will give you a hint https://github.com/anitab-org/anitab-org.github.io/pull/82

Reply:
specifically this file https://github.com/anitab-org/anitab-org.github.io/blob/59515850b2fb6699e33f53148eea401856687d45/src/content/sponsors.js please note those files has been added

Reply:
Sure will make changes here and review.

Reply:
@annabauza, I understood I've to add relative images and how to go about it, adding them to js file. But I couldn't find any images for each project. And should I remove the zulip field and keep only the Github one under "badges"